dev/cosbench-controller-web/src/com/intel/cosbench/controller/web/PrometheusController.java

```diff
@@ -1,3 +1,20 @@
+/** 
+ 
+Copyright 2019 OpenIO, All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Prefer GNU Affero General Public License v3.0

Reply:
I used the same as the others and not a specialist, 

no compatibility problem to use AGPLv3 instead of APACHEv2 ?

Reply:
I think @jfsmig  would prefer the AGPLv3

Reply:
vu avec @jfsmig on laisse les banner existant et on rajoute juste la référence à OpenIO